src/Campaigns/resources/admin/components/CampaignFormModal/index.tsx

```diff
@@ -274,25 +316,55 @@ export default function CampaignFormModal({isOpen, handleClose, apiSettings, tit
                                     register={register}
                                 />
                                 <GoalTypeOption
-                                    type={'donation'}
-                                    label={__('Number of Donations', 'give')}
+                                    type={'donations'}
+                                    label={__('Number of donations', 'give')}
                                     description={__(
                                         'Your goal progress is measured by the number of donations. eg. 1 of 5 donations.',
                                         'give'
                                     )}
-                                    selected={selectedGoalType === 'donation'}
+                                    selected={selectedGoalType === 'donations'}
                                     register={register}
                                 />
                                 <GoalTypeOption
                                     type={'donors'}
-                                    label={__('Number of Donors', 'give')}
+                                    label={__('Number of donors', 'give')}
                                     description={__(
                                         'Your goal progress is measured by the number of donors. eg. 10 of 50 donors have given.',
                                         'give'
                                     )}
                                     selected={selectedGoalType === 'donors'}
                                     register={register}
                                 />
+                                <GoalTypeOption
```

Comment:
Same here. I'd expect these to be visible only if the recurring add-on is installed.

Reply:
Resolved in 373be0c05a118d99af9801c02e4d66a97e196425